src/logger.c

```diff
@@ -151,8 +150,9 @@ void logger_task(void *pv_params) {
       if (client_fd < 0) {
         continue;
       }
-      build_log_string(&log);
-      ssize_t bytes_sent = send(client_fd, &log.message, strlen(log.message), 0);
+      char buffer[MAX_LOGGING_LINE_LEN];
```

Comment:
`char buffer[MAX_LOGGING_LINE_LEN] = {0}`
Zero initialization for char buffer array if you want